app/views/layouts/_navbar.haml

```diff
@@ -20,13 +20,15 @@
         = link_to 'Add New Passenger', new_passenger_path, class: 'nav-link'
       %li.nav-item
         = link_to 'Dispatch Log', log_index_path, class: 'nav-link'
-    - if @current_user&.admin?
+    - if @current_user
```

Comment:
Because passengers aren't users, `@current_user` is `nil` when a passenger is logged in. This seem like a strange choice to me.

Reply:
If there's something actually wrong with how this is done, feel free to make an issue about it. Otherwise, the time to comment on it would have been the PR where it was added.

Reply:
I wouldn't say anything is wrong, it just confused me. I guess I just wanted confirmation that I had the right idea.

Reply:
It _is_ different from our other apps, good on you for figuring out what was going on.